server/boot/authentication.js

```diff
@@ -38,9 +38,33 @@ module.exports = function enableAuthentication(app) {
   ifUserRedirect,
   (req, res) => res.redirect(301, '/auth/auth0'));
 
+  router.get(
+    '/update-email',
+    ifNoUserRedirectHome,
+    (req, res) => res.render('account/update-email', {
+      title: 'Update your email'
+    })
+  );
+
   router.get('/signout', (req, res) => {
     req.logout();
-    res.redirect('/');
+    req.session.destroy( (err) => {
```

Comment:
I think we should be removing all `AccessToken` documents related to the users id also

Reply:
I would prefer that to be done on the mongodb side, using indexes.

Reply:
That's because a search call can be expensive. And we want to avoid that. A user can be logged in from multiple devices and multiple browsers. Looking for which session to invalidate, express side it yet another call to the DB that I think we should avoid.

The bad thing is the DB bloats up. But even that is manageable now, and with indexes they should be optimised?

What do you think?